crates/bevy_internal/src/default_plugins.rs

```diff
@@ -109,7 +110,9 @@ impl PluginGroup for DefaultPlugins {
 
         #[cfg(feature = "bevy_pbr")]
         {
-            group = group.add(bevy_pbr::PbrPlugin::default());
+            group = group
+                .add(bevy_pbr::PbrPlugin::default())
+                .add(PBRDeferredLightingPlugin::default());
```

Comment:
If it were a sub plugin of the `PbrPlugin` will users still be able to bypass it?

Reply:
I guess it could be an option on the PbrPlugin. So `struct PbrPlugin { enable_deferred: bool }` and then in the plugin build() function or so, check its value.